airflow/example_dags/example_twitter_dag.py

```diff
@@ -0,0 +1,152 @@
+#-------------------------------------------------------------------------------------------------------------------------------------
+# Caveat: This Dag will not run because of missing scripts.
+#         The purpose of this is to give you a sample of a real world example DAG!
+#-------------------------------------------------------------------------------------------------------------------------------------
+
+#-------------------------------------------------------------------------------------------------------------------------------------
+# Load The Dependencies
+#-------------------------------------------------------------------------------------------------------------------------------------
+
+import sys
+reload(sys)
+sys.setdefaultencoding("utf-8")
+import time
+from airflow import DAG
+from airflow.operators import BashOperator
+from airflow.operators import HiveOperator
+from airflow.operators import PythonOperator
+from datetime import datetime, timedelta
+from airflow.hooks import HiveCliHook
+from datetime import date, timedelta
+
+#-------------------------------------------------------------------------------------------------------------------------------------
+# Import Scripts for Python callable
+#-------------------------------------------------------------------------------------------------------------------------------------
+
+sys.path.append("python_scripts/example_twitter/")
```

Comment:
it's usually pretty bad to mess with `sys.path`, use relative or absolute imports